src/core/StelUtils.cpp

```diff
@@ -1776,7 +1776,7 @@ double getDeltaTByReingoldDershowitz(const double jDay)
 	}
 	else if ((year >= 1800) && (year <= 1986))
 	{
-		double c = (getFixedFromGregorian(1900, 1, 1)-getFixedFromGregorian(year, 7, 1))/36525.;
+		const double c = (getFixedFromGregorian(year, 7, 1)-getFixedFromGregorian(1900, 1, 1))/36525.;
```

Comment:
Oops… it was my fail :(

Reply:
Can happen, but always unfortunate. Finally fixed.